metals/src/main/scala/scala/meta/internal/bsp/BspConnector.scala

```diff
@@ -282,14 +281,8 @@ class BspConnector(
                   args => bspConfigGenerator.runUnconditionally(buildTool, args)
                 )
                 .map(status => handleGenerationStatus(buildTool, status))
-            case Right(connectionDetails)
```

Comment:
I think this _may_ have been what @dos65 was hinting at in my previous pr, but I realized this also isn't needed here.

Reply:
Yep, that's it!